internal/olm/operator/registry/operator_installer.go

```diff
@@ -54,19 +54,10 @@ func (o OperatorInstaller) InstallOperator(ctx context.Context) (*v1alpha1.Clust
 		return nil, fmt.Errorf("create catalog: %v", err)
 	}
 
-	// TODO: OLM doesn't appear to propagate the "READY" connection status to the catalogsource in a timely manner
```

Comment:
I don't see where this problem was fixed, since this delay occurs for `run packagemanifests` and none of the changes in this PR affect that command's logic.

Reply:
Is this reproducible locally? What is it about `run packagemanifests` that causes the delay?

Also have we initiated a conversation with OLM about this or filed a bug?

Reply:
Will try to reproduce this locally and create a follow-up that basically just uncomments wait function without breaking any of the packagemanifests tests. For now, leaving it commented out.